internal/venice-common/src/main/java/com/linkedin/venice/meta/VersionImpl.java

```diff
@@ -106,6 +106,12 @@ public void setAge(Duration age) {
     this.storeVersion.createdTime = System.currentTimeMillis() - age.toMillis();
   }
 
+  @JsonIgnore
+  @Override
+  public void setCreatedTime(Duration age) {
+    this.storeVersion.createdTime = age.toMillis();
+  }
+
```

Comment:
Its used it a marker to prevent true backup version deletion and piggybacking on an existing field for that. Using a new field would be cleaner but seems like an overkill.